lib/ansible/module_utils/ansible_tower.py

```diff
@@ -30,13 +30,15 @@
 
 try:
     import tower_cli.utils.exceptions as exc
-    from tower_cli.utils import parser
     from tower_cli.api import client
+    from tower_cli.conf import user_dir, CONFIG_FILENAME, Settings
```

Comment:
Doesn't look like `CONFIG_FILENAME` was ever used.

But again, I want to say that I would really like to minimize the amount of imports from tower-cli being made, because those have to be supported indefinitely and they become effectively impossible to ever change.

Additionally, _if_ there was ever a problem with the contract between Ansible and tower-cli, then it will fall into the erroneous `HAS_TOWER_CLI = False` case, which makes debugging a major chore.

Reply:
> Doesn't look like CONFIG_FILENAME was ever used.

Neither was `user_dir`, updated.

> But again, I want to say that I would really like to minimize the amount of imports from tower-cli being made, because those have to be supported indefinitely and they become effectively impossible to ever change.

`tower_cli.parser.string_to_dict` was used and isn't anymore, `tower_cli.conf.Settings` is already imported by modules. The amount of imports from tower-cli doesn't increase.